Packs/ServiceNow/ReleaseNotes/2_5_43.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### ServiceNow v2
+- Updated the Docker image to: *demisto/python3:3.10.13.75921*.
+- Added the ability to pass a custom API root to the ***servicenow-generic-api-call*** command.
```

Comment:
```suggestion
- Added the argument *custom_api* to the ***servicenow-generic-api-call*** command. (please add the explanation of functionality here)
```
please refer to https://xsoar.pan.dev/docs/documentation/release-notes#examples-and-best-practices for xsoar release notes best practices.